documentation/developer_setup.md

```diff
@@ -1,88 +1,74 @@
 # Developer Set Up
 
 * [Installing Go](#installing-go)
-* [Create a Client](#create-a-client)
-	* [Documenting Code](#documenting-code)
-	* [Constructors](#constructors)
-	* [Defining Methods](#defining-methods)
+* [Directory Structure](#directory-structure)
+* [Module Skeletop](#create-module-skeleton)
+* [Create SDK](#create-your-sdk)
 * [Write Tests](#write-tests)
 
 ## Installing Go
 
-The Azure-sdk-for-go team supports Go versions latest and latest-1, to see the exact versions we support you can check the pipeline defintions [here][pipeline_definitions]. The CI pipelines test the latest and latest-1 versions on both Windows and Linux virtual machines. If you do not already have Go installed, refer to this [workspace setup][workspace_setup] article for a more in depth tutorial on setting up your Go environment (there is also an MSI if you are developing on Windows at the [go download page][go_download]). After installing Go and configuring your workspace, fork the `azure-sdk-for-go` repository and clone it to a directory that looks like: `<GO HOME>/src/github.com/Azure/azure-sdk-for-go`.
+The Azure SDK for Go supports the latest two versions of Go. When setting up a new development environment, we recommend installing the latest version of Go per the [Go download page][go_download].
 
+### Configuring VSCode
 
-## Create a Client
+If you're using VSCode, install the Go extension for VSCode. This usually happens automatically when opening a .go file for the first time.
```

Comment:
Recommend running VSCode with WSL. The debugger is slow otherwise. In addition to installing the Go extension, you'll have to install the VSCode Go tools. There will be a notification reminding you to install them in VSCode or can be done through the VSCode command Go: Install/Update Tools

Reply:
I'm hesitant to include any info about this as I want to keep it short to just the requirements.

For the record, I've struggled with this too and think it's AV nonsense getting in the way.